### PR TITLE
New Automation to add BZ 2247187 client evict defer workflow

### PR DESCRIPTION
**Pull request by sumabai**
# Description
[BZ 2247187](https://bugzilla.redhat.com/show_bug.cgi?id=2247187) workflow : Defer client evict if OSD is laggy

Polarion TC : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581592

Jira ID : https://issues.redhat.com/browse/RHCEPHQE-12598

Test suite : suites/reef/cephfs/tier-2_cephfs_test-clients.yaml
- test:
      abort-on-fail: false
      desc: "Verify Client eviction is deferred if OSD was laggy"
      module: cephfs_bugs.test_defer_client_evict_on_laggy_osd.py
      name: Client eviction deferred if OSD is laggy
      polarion-id: "CEPH-83581592"

Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UZFXJW

Added another BZ workflow BZ2238663 to the same PR.

Polarion TC : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581613
Jira ID : https://issues.redhat.com/browse/RHCEPHQE-12598
Test suite : suites/reef/cephfs/tier-2_cephfs_test-clients.yaml
- test:
      abort-on-fail: false
      desc: "Verify Client is blocklisted if session metadata is bloated"
      module: cephfs_bugs.test_client_blocklist_large_session_metadata.py
      name: Verify Client is blocklisted if session metadata is bloated
      polarion-id: "CEPH-83581613"

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5OHZJT

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_bugs/test_defer_client_evict_on_laggy_osd.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    except CommandFailed as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        client1.exec_command(sudo=True, cmd=f"umount {fuse_mount_dir}", check_ec=False)
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_bugs/test_defer_client_evict_on_laggy_osd.py:211]: Do we need to set this value back to true `defer_client_eviction_on_laggy_osds`

**Review by neha-gangadhar — COMMENTED** [review]
lgtm

File: tests/cephfs/cephfs_bugs/test_client_blocklist_large_session_metadata.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        fs_name = "cephfs"
+        log.info("Enable ceph logging for ceph client")
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd="ceph config set client log_to_file true",
+        )
+        log.info("Set client config client inject fixed oldest tid to true")
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd='ceph config set client "client inject fixed oldest tid" true',
```
Comment (hkadam134) [inline tests/cephfs/cephfs_bugs/test_client_blocklist_large_session_metadata.py:52]: Can we validate the values after setting ? using 'ceph config get'

File: tests/cephfs/cephfs_bugs/test_client_blocklist_large_session_metadata.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        )
+        log.info("Set mds config mds_max_completed_requests 10000")
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd="ceph config set mds mds_max_completed_requests 10000",
```
Comment (hkadam134) [inline tests/cephfs/cephfs_bugs/test_client_blocklist_large_session_metadata.py:57]: Can we validate the values after setting ? using 'ceph config get'

**Review by hkadam134 — APPROVED** [review]
Minor comments, rest all looks good.

**Review by hkadam134 — APPROVED** [review]
Looks Good

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**